examples/simple/index.nobuild.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Pure Reason Example</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <script>
+    window.renderIcon = "⧉";
+    window.eventIcon = "⚡";
+    window.propsReceivedIcon = "🠫";
+    window.updateIcon = "♼";
+    window.elbowIcon = "⮡";
+  </script>
+</head>
+<body style="">
+  <div id="index"></div>
+  <!-- <script src="../../bundledOutputs/simple.js"></script> -->
+  <script src="./console.style.js"></script>
+  <script data-main="../../lib/js/src/simple/simpleRoot.js" data-project-root="../../" src="../require_polyfill.js">
```

Comment:
Oh I just copied things over lol.

I might move these arounds though. Otherwise it might be confusing.